annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/transcribe/BaseTranscribingTest.java

```diff
@@ -10,6 +10,7 @@
 
 	public static final String TRANSCRIPTION_MINIMAL = "/transcription/minimal.json";
```

Comment:
must update the minimal template with the mandatory fields.
The json templates for the create annotation method must be updated after making the fields mandatory.
The jsp template must indicate the mandatory fields

Reply:
Added mandatory fields edmRights and value to minimal json template. JSP template documentation extended with description of mandatory fields.